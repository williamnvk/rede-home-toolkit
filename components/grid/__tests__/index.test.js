import React from "react"
import { shallow } from 'enzyme';
import { Column, Row } from '../index.js';

describe('Column', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Column />);
    expect(wrapper).toMatchSnapshot();
  });
});


describe('Row', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Row />);
    expect(wrapper).toMatchSnapshot();
  });
});
