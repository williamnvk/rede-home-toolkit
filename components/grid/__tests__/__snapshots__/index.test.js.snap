// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders correctly 1`] = `ShallowWrapper {}`;

exports[`Row renders correctly 1`] = `ShallowWrapper {}`;
